com.epicgames.ut99.metainfo.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop-application">
+  <id>com.epicgames.ut99</id>
+  <launchable type="desktop-id">com.epicgames.ut99.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
```

Comment:
Add developer_name

Reply:
It already is [here](https://github.com/flathub/flathub/pull/4599/files#diff-eda253a221abd600482a95b3a8bc06da77f052fb55a91c477f5a07f4ff37d3fdR30). :-)

Reply:
😅